.github/actions/publish-nuget/action.yml

```diff
@@ -199,10 +199,13 @@ runs:
         ${{ inputs.dotnet-pack-ignore-warning }}
         '@.Split() | ? { $_ }
 
+        $version = '${{ steps.setup.outputs.publish-version }}'
+        $enablePackageValidation = !($version -like '*-*')
```

Comment:
My impression is, this conditional package validation can easily lead to confusion.
I know changing the workflow parameter on the issue branch before publishing a prerelease version does not improve the developer experience, but please consider exposing this as a script parameter.

Reply:
I assume you mean action (and workflow?) parameter, since it's already a script parameter. How would that work, though? Would it override this alpha detection logic (we can have the default for it as empty string to allow this)?

We need this logic in any case, since not only has this been the behavior for a very long time, but it's also useful, since we never need package validation for alphas.

Reply:
> ..., but it's also useful, since we never need package validation for alphas.
If it's really not necessary, then it's okay. Previously, it was always validated, which is why I have a bad feeling here.